aviary/docs/user_guide/aviary_commands.ipynb

```diff
@@ -143,9 +158,14 @@
     "\n",
     "The `aviary fortran_to_aviary` command will convert a Fortran input deck to an Aviary csv.\n",
     "\n",
-    "The only required input is the filepath to the input deck.\n",
-    "Optionally, a deck of default values can be specified, this is useful if an input deck assumes certain values for any unspecified variables.\n",
+    "The only two required inputs are `-l` (for `--legacy_code` with options `FLOPS` and `GASP`) and the filepath to the input deck. \n",
+    "Optionally, a deck of default values can be specified via the option `-d` (for `--defaults_deck`) and a default deck file, this is useful if an input deck assumes certain values for any unspecified variables.\n",
+    "When this command is run, a brief message is printed. To print more messages, one can set verbosity level higher. For example, `-v 3` will result in debug messages being printed.\n",
```

Comment:
add a link to the verbosity page here (`Controlling Display Levels` in developer_guide/coding_standards.ipynb)

Reply:
Done.